doc/en/overview.rst

```diff
@@ -1,13 +0,0 @@
-==================================================
```

Comment:
Those were all moved to the main "contents.rst"... having this in a nested TOC didn't seem to bring much to the table. All those documents can  be found directly in the contents page now.